common/kokkos-sampler/kp_sampler_skip.cpp

```diff
@@ -27,6 +31,45 @@ static endFunction endForCallee                = NULL;
 static endFunction endScanCallee               = NULL;
 static endFunction endReduceCallee             = NULL;
 
+void getGlobFenceChoice() {
```

Comment:
The style doesn't match the rest of the file.
```suggestion
void get_global_fence_choice() {
```